R/mse_estimation.R

```diff
@@ -143,19 +147,30 @@ mse_estim <- function(framework,
     framework$threshold <-
       framework$threshold(y = pop_income_vector)
   }
+
+  if(is.null(framework$aggregate_to_vec) != TRUE){
```

Comment:
I do not think that you need to compare to TRUE, should be possible with !is.null(), right? And I am still not sure if I get that you check if aggregate_to is null but that you do not need to check if pop_weights are null. But we already had this topic and when you are sure that it is correct, I trust on that ;)